### DISCUSSION TITLE
How to publish notes with mathbooster callouts on Quartz?

### DISCUSSION THREAD

**Discussion by mcrco** [RyotaUshio/obsidian-latex-theorem-equation-referencer#219] [Q&A] [answered]
Hi guys,

I'm trying to publish my obsidian notes (with mathbooster environments/callouts) to Quartz. Does anyone know how I can get the custom mathbooster callouts to work? Right now they are simply being rendered as blockquotes.

Thanks!

**Comment by RyotaUshio** [comment, marked as answer]
I'm not aware of ways of reliably exporting theorem callouts to quartz. But you can use the [webpage HTML export](https://github.com/KosmosisDire/obsidian-webpage-export) plugin to export Obsidian notes containing theorem callouts to HTML files while preserving the theorem styling.
The documentation website of this plugin also uses the plugin.
https://ryotaushio.github.io/obsidian-latex-theorem-equation-referencer/theorem-callouts/theorem-callouts.html

**Reply by mcrco** [reply]
ooo this looks perfect, I was just gonna ask how the docs are published. thanks!
